src/Controller/TokenController.php

```diff
@@ -80,4 +81,35 @@ public function getToken(string $uniqueId): JsonResponse
 
         return new JsonResponse(['token' => $token->getToken(), 'expire' => $token->getTokenExpires()]);
     }
+
+    /**
+     * @Route("/token/refresh", name="refresh_token", methods={"POST"})
+     *
+     * @param request $request
+     *   The request
+     *
+     * @return JsonResponse
+     *
+     * @throws \Exception
+     */
+    public function refreshToken(Request $request): JsonResponse
+    {
+        $body = json_decode($request->getContent());
+
+        $token = $body->token;
```

Comment:
Will this throw an exception if token is in the body from json_decode?

Reply:
Fixed